domene/src/main/java/no/nav/foreldrepenger/journalføring/ManuellOpprettSakValidator.java

```diff
@@ -45,6 +46,15 @@ public void validerKonsistensMedSak(JournalpostId journalpostId, FagsakYtelseTyp
         requireNonNull(aktørId, "Ugyldig input: AktørId kan ikke være null ved opprettelse av en sak.");
 
         var arkivJournalpost = arkivTjeneste.hentArkivJournalpost(journalpostId.getVerdi());
+        validerKonsistensMedSak(arkivJournalpost, oppgittFagsakYtelseTypeDto, aktørId, nyDokumentTypeId);
+    }
+
+    public void validerKonsistensMedSak(ArkivJournalpost arkivJournalpost, FagsakYtelseTypeDto oppgittFagsakYtelseTypeDto, AktørId aktørId,
```

Comment:
Er det noe sted vi sjekker at aktørid på journalpost og aktørid på sak er samme? Burde vi i såfall det?

Reply:
Det bør håndteres lenger opp, ikke her. Man bruker aktørId fra saken som den bestemmende.
Normalt havner innkommende på en sak som tilhører angitt bruker eller det opprettes ny sak for denne.
Journalføringsoppgaver lar deg velge saker som tilhører bruker.
Knytt til annen sak - det samme utvalget av saker.

Mulig det er behov for å endre bruker i enkelte tilfelle - Det kan være at dokumentet er sendt inn for en part, men også skal inn på saken til en annen part. Typisk dersom avsender er en tredjepart (advokat, sykehus, osv).